ShopApp/App/Extensions/UIViewController+Alert.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  UIViewController+Alert.swift
+//  ShopApp
+//
+//  Created by Evgeniy Antonov on 4/23/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import UIKit
+
+private let kQuantityCountMax = 3
+
+extension UIViewController {    
```

Comment:
Is it possible to write tests for this extension?

Reply:
We can't get presented alert controller in tests